app/validators/schema_validator.py

```diff
@@ -41,7 +41,8 @@ def validate(self):
             return {}
         except ValidationError as e:
             match = best_match([e])
-            self.add_error(match.message, verbose=e.message, path=e.path)
+            path = "/".join([str(path_element) for path_element in e.path])
```

Comment:
Minor, but the list isn't required.
```suggestion
            path = "/".join(str(path_element) for path_element in e.path)
```